src/main/java/mate/academy/validation/email/Email.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.validation.email;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import jakarta.validation.constraints.NotBlank;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+@NotBlank
```

Comment:
double check whether you need to use `@ConstraintComposition` in order to include `@NotBlank`

or perhaps you can remove `@NotBlank` at all and let the regex validate everything

Reply:
removed @NotBlank